features/steps/steps.py

```diff
@@ -1,20 +1,27 @@
+import os
 import ast
 import json
 import typing
 import operator
 import functools
+import itertools
+import csv
 import re
 import itertools
 import math
 
 from collections import Counter
 from dataclasses import dataclass, field
+from pathlib import Path
+from parse_type import TypeBuilder
 
 import ifcopenshell
 import pyparsing
 
 from behave import *
 
+register_type(from_to=TypeBuilder.make_enum({"from": 0, "to": 1 }))
+register_type(maybe_and_following_that=TypeBuilder.make_enum({"": 0, "and following that": 1, "and return to first": 2 }))
```

Comment:
name of `maybe_and_following_that` should be changed, but I guess in my gherkin proposal we wouldn't have another tail option.